manage.py

```diff
@@ -51,19 +46,14 @@ def test(all=False):
 @manager.command
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
-    iterations = 0
     categorizer = Categorizer()
 
     while True:
         logging.info("Starting scrape...")
-        if (iterations % ITERATIONS_BEFORE_RESET) == 0:
-            categorizer.reset_title_categories()
 
         scrape_articles(categorizer)
 
-        iterations += 1
-        logging.info("Finished scrape. Zzz...")
-        time.sleep(SCRAPE_INTERVAL)
+        logging.info("Finished scrape. Let's run it back...")
```

Comment:
I guess not. Mainly because it requires more cycles when it's running than when it's sleeping, but if you say so. It's only good not to sleep if the time to categorize new articles equals the time to scrape and categorize the previous set of new articles.